css/types/color.json

```
@@ -194,6 +194,69 @@
             }
           }
         },
+        "color_mix": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/color_value/color_mix()",
```

I couldn't find an issue or PR for this on the content side of things. I'd guess that you're planning to use the actual name from the spec, but feel free to ignore this suggestion if not.

```suggestion
            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/color_value/color-mix()",
```